lib/lita/handlers/lunch_reminder.rb

```diff
@@ -180,17 +180,44 @@ def load_on_start(_payload)
       end
 
       route(/.*/i, command: false) do |response|
-        if quiet_time? && Lita::Room.find_by_name("lita-test").id == response.room.id
-          user = Lita::User.find_by_mention_name("agustin")
-          message = "Estoy empezando a sugerir que evitemos hablar en #coffeebar entre las 10 y las " \
-          "12 del día para que la gente en la oficina pueda concentrarse. Las interrupciones" \
-          " hacen muy dificil trabajar! mira: http://www.paulgraham.com/makersschedule.html"
+        if quiet_time? && Lita::Room.find_by_name('lita-test').id == response.room.id
+          user = Lita::User.find_by_mention_name('agustin')
+          message = 'Estoy empezando a sugerir que evitemos hablar en #coffeebar entre las 10 y las ' \
+          '12 del día para que la gente en la oficina pueda concentrarse. Las interrupciones' \
+          ' hacen muy dificil trabajar! mira: http://www.paulgraham.com/makersschedule.html'
           robot.send_message(Source.new(user: user), message) if user
         end
       end
 
-      def broadcast_to_audit_channel(message)
-        target = Source.new(room: '#karma-audit')
+      route(/vend[oe] (mi|\s)? ?almuerzo/i, command: true) do |response|
+        user = response.user
+        order = create_order(user, 'limit')
+        unless @market.add_limit_order(order)
+          response.reply('No puedes vender algo que no tienes!')
+          next
+        end
+        response.reply_privately(
+          "@#{user.mention_name}, tengo tu almuerzo en venta!"
+        )
+        broadcast_to_channel("@#{user.mention_name}, tengo tu almuerzo en venta!", '#cooking')
+      end
+
+      route(/c(o|ó)mpr(o|ame|a)? (un )?almuerzo/i, command: true) do |response|
+        user = response.user
+        order = @market.add_market_order(user.id)
+        unless order
+          response.reply('no te puedo comprar almuerzo...')
+          next
+        end
+        seller_user = Lita::User.find_by_id(order['user_id'])
+        response.reply_privately("@#{user.mention_name}, ya te consegui almuerzo!")
+        broadcast_to_channel(
+          "@#{user.mention_name} le compró almuerzo a @#{seller_user.mention_name}",
+          '#cooking')
```

Comment:
Closing method call brace must be on the line after the last argument when opening brace is on a separate line from the first argument.